chart/operator/templates/k8sgpt-crd.yaml

```diff
@@ -212,3 +212,5 @@ status:
     plural: ""
   conditions: []
   storedVersions: []
+
+
```

Comment:
2 new lines?

Reply:
I think this is because I copied it over manually, ill remove